### PR TITLE
Correcting Civo urls to use www

### PR DESCRIPTION
**Pull request by civo-claire**
Fixing non www. urls as well as updating the sign up link to dashboard link

### CODE REVIEW

File: profile/README.md [hunk 1]
```diff
@@ -3,9 +3,9 @@ Welcome to Civo on GitHub</h3>
 <p align="center">
   <img src="https://gpvc.arturio.dev/civo" alt="profile views"> •  
   <a href="https://twitter.com/intent/follow?screen_name=CivoCloud&tw_p=followbutton"><img src="https://img.shields.io/twitter/follow/CivoCloud?label=%40CivoCloud&style=social"></a>  •
-  <a href="https://civo.com">Website</a> •
+  <a href="https://wwww.civo.com">Website</a> •
```
Comment (kaihoffman) [inline profile/README.md]: This should be `www.`

Comment (kaihoffman) [reply profile/README.md]: Otherwise looks good!

**Review by kaihoffman — CHANGES_REQUESTED** [review]

File: profile/README.md (hunk 1 above)
Comment (kaihoffman) [reply profile/README.md]: @civo-claire  just checking if you want to push the above change to merge

Comment (civo-claire) [reply profile/README.md]: Yes please @kaihoffman

**Review by kaihoffman — APPROVED** [review]
Approving

**Pull request merged**